heron_base/launch/base.launch

```diff
@@ -26,26 +26,40 @@
         <node ns="wireless" pkg="wireless_watcher" name="wireless_watcher" type="watcher_node" respawn="true" />
     </group>
 
-    <group if="$(eval arg('namespace') == '')">
-     <node pkg="um6" type="um6_driver" name="um6_driver" output="screen">
-       <param name="tf_ned_to_enu" value="true"/>
-       <param name="/um6_driver/port" value="$(optenv HERON_IMU_UM6_PORT /dev/clearpath/imu)" />
-       <remap from="imu/data" to="imu/data_raw" />
-       <param name="lt_model" value="true" />
-       <param name="linear_acceleration_stdev" value="0.38" />
-       <param name="frame_id" value="imu_link" />
-     </node>
+    <group unless="$(optenv HERON_IMU_UM6 0)">
+      <!-- newer herons use the microstrain CM5-25 IMU -->
```

Comment:
Should there be a case if there is a namespace?

Reply:
Possibly. I'm looking into that along with the 2 points below

Reply:
Added the namespace case, checked the topics & default device as mentioned in the other notes.